src/main/java/com/google/api/codegen/transformer/ruby/RubyGapicSurfaceDocTransformer.java

```diff
@@ -44,10 +44,14 @@
 import com.google.protobuf.Api;
 import java.io.File;
 import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class RubyGapicSurfaceDocTransformer implements ModelToViewTransformer {
   private static final String DOC_TEMPLATE_FILENAME = "ruby/message.snip";
 
+  private static final Pattern SENTENCE_PATTERN = Pattern.compile("^([^.]+)\\.(?: |$)");
```

Comment:
Consider handling `\.\w`:

"This is an API for that turns `google.longrunning.Operations` instances into kittens."